### ISSUE TITLE
Improve tools for speck checking

### ISSUE THREAD

**Issue by amahboubi** [math-comp/mcb#14]
See the suggestions by @mdnahas in the (merged) #11.

**Comment by mdnahas** [comment]
If this is done, the "make spell" should probably be moved into the Makefile in the tex/ directory.  I didn't notice it when I wrote the code.

**Comment by amahboubi** [comment]
Yes, I noticed that when testing. But it's no big deal. In fact, as the README says, the doc/ directory is meant for the material put on the webpage of the book.

**Comment by amahboubi** [comment]
I did my best to address this issue with commit  74f281a, considering my (low) technical skills. See the commit log. In particular I did nothing w.r.t. the suggesting to use git hooks. I will not have more time to devote to this so I will close this issue if on one reacts within the next week or so.
